src/Tribe/Tickets_Handler.php

```diff
@@ -1174,10 +1182,23 @@ public function get_ticket_max_purchase( $ticket_id ) {
 			return 0;
 		}
 
-		/** @var Tribe__Tickets__Ticket_Object $ticket */
 		$ticket = $provider->get_ticket( $event, $ticket_id );
 
-		$available = $ticket->available();
+		if ( ! $ticket instanceof Tribe__Tickets__Ticket_Object ) {
+			return 0;
+		}
+
+		$max_at_a_time = $this->get_max_qty_allowed_to_be_added_to_cart_at_a_time( $ticket );
+
+		// The actual ticket stock, not limited by Max At A Time
+		$stock_available = $ticket->available();
+
+		// change Unlimited to Max At A Time
```

Comment:
:no_entry_sign: Inline comments must end in full-stops, exclamation marks, or question marks

<sub>hash: 3e66f94e54ddf524ba026a11a4e57fe8</sub>